WalletWasabi.Documentation/BackendDeployment.md

```diff
@@ -4,21 +4,43 @@ Consider updating the versions in `WalletWasabi.Helpers.Constants`. If the versi
 
 ```sh
 sudo apt-get update && cd ~/WalletWasabi && git pull && cd ~/WalletWasabi/WalletWasabi.Backend && dotnet restore && cd ~
+
+# Stop
 sudo service nginx stop
 sudo systemctl stop walletwasabi.service
 sudo killall tor
 bitcoin-cli stop
+
+# Status checks
+systemctl is-active tor && systemctl is-active walletwasabi && systemctl is-active nginx && ps -C bitcoind >/dev/null && echo "active" || echo "incative"
```

Comment:
```suggestion
systemctl is-active tor && systemctl is-active walletwasabi && systemctl is-active nginx && ps -C bitcoind >/dev/null && echo "active" || echo "inactive"
```

Reply:
Would it make sense to have 4 checks like:

```bash
systemctl is-active tor && echo "Tor: Running" || echo "Tor: NOT RUNNING!"
```

to clearly see what is running and what is not running?

Reply:
Maybe write a few short scripts as this is becoming tedious.

BTW you could abuse echo + systemctl:

```bash
echo -n 'Tor: '; systemctl is-active tor; echo -n 'WalletWasabi: '; systemctl is-active walletwasabi; echo -n 'Nginx: '; systemctl is-active nginx;
```

This will print (in success case):

```
Tor: active
WalletWasabi: active
Nginx: active
```

`bitcoind` needs to be handled differently obviously.

Reply:
> Maybe write a few short scripts as this is becoming tedious.

Or write a Bash function, possibly. :)

Reply:
Guys, one line, only one line :)

Reply:
Ok I have updated this. Removed nginx.